### ISSUE TITLE
Collect PMA_HOSTS via PuppetDB

### ISSUE THREAD

**Issue by jacksgt** [vision-it/vision-phpmyadmin#7]
It would be nice if we collected the PMA_HOSTS via our Puppet Module vision_mysql when installing the server (like we did before), instead of hardcoding them into hiera (which is prone to be forgotten).

https://github.com/vision-it/vision-intranet/blob/703ac1f6463669c48733829c7e5066bf5cbe82d7/manifests/database.pp#L35

**Comment by martialblog** [comment]
We can do that with https://github.com/dalen/puppet-puppetdbquery

**Comment by jacksgt** [comment]
> You might not need this puppet module anymore. PuppetDB bundles a simplified query language since version 4.0. So unless you really prefer the syntax in this module you can just use PQL instead. See https://puppet.com/blog/introducing-puppet-query-language-pql for more details.

https://github.com/dalen/puppet-puppetdbquery#usage-warning

We can still use exported facts (despite masterless setup)

**Comment by martialblog** [comment]
Yes, I saw that... thought we use the synergy since we're already using the module: https://github.com/vision-it/vision-icinga2/blob/production/.fixtures.yml

**Comment by jacksgt** [comment]
Why not just like so?

https://github.com/vision-it/vision-xen/blob/c10357789a9194d2c216caede201ce8d8d61059c/manifests/hostsfile.pp#L25

```
Host <<| tag == $dom0_hostname |>> {
    notify => Service['dnsmasq'],
}
```

**Comment by martialblog** [comment]
We don't need a "resource", just an array of hostnames and human-readable names:

```
vision_phpmyadmin::pma_hosts:
  - 'foobar.com'
vision_phpmyadmin::pma_verboses:
- 'barfoo'
```

So we can't use exported resources...

**Comment by jacksgt** [comment]
I mean I haven't tried it, but it does seem like that is possible with exported resources:
https://puppet.com/docs/puppet/5.3/lang_exported.html#syntax
